### PR TITLE
Bump version to 1.2.0-preview.

### PR DESCRIPTION
**Pull request by vincent-unity**
Prepare a new 1.2.0 release:
1. Improved notification setting UI change
2. Moved the notification setting asset under 'Project Settings'
3. Fixed some bugs on iOS and Android.

QA: Please verify the il2cpp build on Android as we removed the long[] to int[] workaround for vibration patterns.

### CODE REVIEW

**Comment by vincent-unity** [comment]
@lucianUnity3D FYI.

File: com.unity.mobile.notifications/CHANGELOG.md [hunk 1]
```diff
@@ -2,10 +2,20 @@
 
 All notable changes to this package will be documented in this file.
 
+## [1.2.0-preview] - 2020-04-16
+
+### Changes & Improvements:
+- [Editor] Improved the Notification Settings UI, also stored the notification setting asset under 'Project Settings' folder rather than 'Assets' folder.
```
Comment (todi1856) [inline com.unity.mobile.notifications/CHANGELOG.md:8]: It's 'ProjectSettings', not 'Project Settings' :)

Comment (vincent-unity) [reply com.unity.mobile.notifications/CHANGELOG.md]: Nice catch. Fixed.

**Comment by vaidasma** [comment]
Works as intended on IL2CPP and Mono (.NET 4.x and .NET Standard 2.0).
The only concern that I have from the UX side of the project settings is that the Notification window doesn't scale well when shrinking the window horizontally (check the right side of the window).  @vincent-unity you think you can implent an easy fix for this? 
image

**Comment by vincent-unity** [comment]
@vaidasma I think the size is limited with minimal 640 at the very beginning, so I didn't change that part but only made the scrollbar work correctly. If you check other settings like Editor & Player, you can also see the scrollbar at the bottom but with a smaller minimal value.

It's not an easy fix but I can do it with https://github.com/Unity-Technologies/com.unity.mobile.notifications/pull/36#discussion_r409251293 in next pr, by changing to a smaller minimal value. How do you think?

**Comment by vaidasma** [comment]
@vincent-unity Okay, that works. I can see other windows, such as Player settings, dealing better with resizing the default layout horizontally by shrinking some of the containers appropriately. Should I make a ticket so it would be easier to remember?

**Comment by vincent-unity** [comment]
@vaidasma Please make a ticket. And I would also like to discuss with UX guys about this to see what the best behavior could be here.

**Pull request merged**